pkg/alerts/alerts_test.go

```diff
@@ -50,11 +50,11 @@ func newIntegrationTestClient(t *testing.T) Alerts {
 		t.Skipf("acceptance testing requires NEW_RELIC_API_KEY and NEW_RELIC_ADMIN_API_KEY")
 	}
 
-	client := New(config.Config{
+	a := New(config.Config{
 		AdminAPIKey:    adminAPIKey,
 		PersonalAPIKey: personalAPIKey,
 		LogLevel:       "debug",
 	})
 
-	return client
+	return a
```

Comment:
Not a blocker at all and not that it matters, but we could return early vs setting to a variable.